### DISCUSSION TITLE
Refined Storage

### DISCUSSION THREAD

**Discussion by chimpman73** [EnigmaticaModpacks/CreateTogether#15] [Q&A] [answered]
I am currently running CreatTogether v1.09.  I watched Xisuma set up a Refined Storage interface to his Storage Drawers in episode 3 of Create Together (around the 32 min mark) https://www.youtube.com/watch?v=SAyNu_4OSvQ&t=1920s

I was wondering if the Refined Storage is meant to have a power source or not?  It looked like when Xisuma placed the Controller it was automatically powered, however when I place the Controller it seems to require an external power source (FE).  I also noticed that the crafting recipes were different back when X made that video, so I'm wondering if maybe something happened when Refined Storage was updated in v1.05. 

Is anyone else able to get the Refined Storage interface to work without a power source?

**Comment by theboo** [comment]
Refined storage does not need power in CT, it sounds like your config and kubejs folders did not download properly when you installed the pack. I would recommend reinstalling the pack..

The power requirement is changed through config, and the recipe changes and unused item hiding is done through kubejs scripts..

You should also only see crafting grid, cable, external storage and controller in jei.. everything else from the mod should be disabled and hidden(they will still show up in creative menu, however)

**Reply by chimpman73** [reply]
Ahh... thanks!  Actually when I run the mod directly through Curse Forge it does seem to behave as expected.  However I'm also running the mod on a linux box as a server, and when I do that, the Controller doesn't behave correctly.

I did double check and there were some files missing on the server - the defaultconfigs dir was empty (this is where the Controller power settings are configured), but even adding those files onto the server didn't solve the problem.

Regardless, you've given me a good starting point to try and debug this problem.  I'll let you know when I figure out what is going on.  Thanks!

**Reply by chimpman73** [reply]
Ahh... and in the world/config dir there is a refined storage config with energy=true.  changing that fixes the problem.

**Comment by theboo** [comment, marked as answer]
Yeah, the server configs in the default config folder are copied over to the serverconfigs folder in your world save at world creation, so if you were missing those it would have just generated the default versions of the configs instead of the edited ones niller made

**Reply by chimpman73** [reply]
Ok, that makes sense now.  I should probably just replace the default configs in the world directory with the ones that niller made.  Thank you!  That was a huge help!
